source/_posts/2017-05-19-kanban-method-in-a-consulting-company.md

```diff
@@ -0,0 +1,123 @@
+which aids decision-making about what, when and how much to produce. Although
+the method (inspired by the Toyota Production System and lean manufacturing) 
+originated in software development and IT, it may be applied to any professional
+service whose work outcome is intangible rather than physical.
+
+
+## Tools
+
+The basic work tools are boards and stickers, but in our company we use visual
+boards in [Trello](https://trello.com/), which streamlines our work in kanban.
+Our boards are divided into appropriate categories: Backlog, To Do, In Progress,
+Verification, Done and Project Status, where we can discuss project features
+or problems.
+
+![1](http://imageshack.com/a/img922/6156/YfxSE1.png)
+
+## Cards (Kanbans)
+
+According to the Kanban method we give out a kanban cards. Every card must have:
+
+* Title - As needed, it can be very general or low-level,
+* Description - With some documentation, links or to do list,
+* Asignee - If you want to plan something with others You can assing more than
+one person, but we recommend to assing one person / per task,
+* Label - According to the task,
+* Due date - We set a maximum of two weeks to complete the tasks.
+
+The card is very important in the whole project process. That way, we measure 
+our work and set time for each stage in the design, develop and testing 
+solution phases in embedded systems projects. Additionally inside the
+card we can discuss the features and see, what happened since last time.
+
+![img](http://imageshack.com/a/img924/2586/spqDrv.png)
+
+## Workflow
+ 
+The card life begins in Backlog, tasks here have lowest priority and they are
+waiting for select to development - but it's not a rule, sometimes we add cards
+with high priority directly to ToDo column. ToDo is the queue for the most
+important column - In Progress. From there our goal is to safely move this task
+to done. By analyzing the boards we are able to find the blocker very quickly.
+In embedded systems company, we are dealing mostly with blockers in the form of
+not working or missing hardware, incompatibility of the kernel with hardware and 
+frequent changes in priorities.
+
+![kanban-flow](http://imageshack.com/a/img923/2471/u20Zzr.jpg)
+
+## Rules
+
+To make kanban work, each employee must follow these rules:
+
+* In Progress has the biggest priority, next is ToDo and Backlog,
+* Never take more than two tasks at a time,
+* Do not throw unfinished tasks for verification, because due date is comming,
+* Listen and read communication, ask questions,
+* Discuss inside cards - about problems and progress,
+* Suggest changes in places where you see such a need.
+
+These rules are only a sample but improve the whole process. We know situations
+where tasks are going fast to verification, after which it turns out that the
+task is incomplete and we need to turn it back to the left. When the checks show
+such errors, we try to first determine the degree of completion of the card, and

```

Comment:
"we try to determine the degree of completion of the card first, and then..." Wrong sentence order